<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GestionPreguntas</class>
 <widget class="QMainWindow" name="GestionPreguntas">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gestión de Preguntas</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="textoLabel">
        <property name="text"><string>Texto de la Pregunta:</string></property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QTextEdit" name="textoTextEdit"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="imagenLabel">
        <property name="text"><string>Imagen:</string></property>
       </widget>
      </item>
      <item row="1" column="1">
       <layout class="QHBoxLayout" name="imagenLayout">
        <item>
         <widget class="QLineEdit" name="imagenLineEdit"/>
        </item>
        <item>
         <widget class="QPushButton" name="browseImagePushButton">
          <property name="text"><string>Browse…</string></property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="opA_Label">
        <property name="text"><string>Opción A:</string></property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="opA_LineEdit"/>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="opB_Label">
        <property name="text"><string>Opción B:</string></property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="opB_LineEdit"/>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="opC_Label">
        <property name="text"><string>Opción C:</string></property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLineEdit" name="opC_LineEdit"/>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="correctaLabel">
        <property name="text"><string>Respuesta Correcta (A, B o C):</string></property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QLineEdit" name="correctaLineEdit"/>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="categoriaLabel">
        <property name="text"><string>Categoría:</string></property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QComboBox" name="categoriaComboBox"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="buttonsLayout">
      <item>
       <widget class="QPushButton" name="crearPushButton">
        <property name="text"><string>Crear</string></property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="listarPreguntasPushButton">
        <property name="text"><string>Listar Preguntas</string></property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTableView" name="preguntasTableView"/>
    </item>
    <item>
     <layout class="QHBoxLayout" name="editarEliminarLayout">
      <item>
       <widget class="QPushButton" name="editarPushButton">
        <property name="text"><string>Editar Pregunta</string></property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="eliminarPushButton">
        <property name="text"><string>Eliminar Pregunta</string></property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar"/>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>